### ISSUE TITLE
Extending the Nuxt examples section with Nuxt routing / transitions

### ISSUE THREAD

**Issue by 4aficiona2** [Gomah/bulmil#44]
I was wondering if it would be possible to extend the Nuxt router, maybe in [nuxt-stencil](https://github.com/Gomah/nuxt-stencil), similar to the following implementation in [Ionic Vue](https://github.com/ModusCreateOrg/ionic-vue) so the Nuxt Router could be used with Stencil?

@see https://github.com/ModusCreateOrg/ionic-vue/blob/master/src/router.ts

**Comment by Gomah** [comment]
Good idea! I think the example should use the [router-module](https://github.com/nuxt-community/router-module) to support `IonicVueRouter` then.

I don't have much time to look at it now, PRs are welcome 👍
